NameLayerMain/src/vg/civcraft/mc/namelayer/gui/GUIGroupOverview.java

```diff
@@ -78,14 +77,14 @@ public void clicked(Player p) {
 					NameLayerPlugin.log(Level.INFO,
 							p.getName() + " turned autoaccept for invites off "
 									+ "via gui");
-					NameLayerPlugin.getGroupManagerDao().removeAutoAcceptGroup(p.getUniqueId());
+					NameLayerPlugin.getGroupManagerDao().removeAutoAcceptGroupAsync(p.getUniqueId());
```

Comment:
Interesting, looking through the code for autoacceptance, it doesn't seem to be cached anywhere. I think until there is a proper cache setup for those, I think this should stay as is.